frappe/core/doctype/report/report.json

```diff
@@ -27,7 +27,7 @@
   "report_script",
   "client_code_section",
   "javascript",
-  "json",
+  "report_json",
```

Comment:
this will require a patch. 

Maybe just keep the fieldname same but don't use `json` as a variable or argument in python. It shouldn't be a problem then.